Jenkinsfile

```diff
@@ -78,7 +78,7 @@ lines.each {line ->
                     sh 'mv megawar-$LINE.war megawar.war && bash pct.sh'
                 }
                 def launchableSession = readFile("launchable-session-${line}.txt").trim()
-                launchable("record tests --session ${launchableSession} maven './**/target/surefire-reports'") // TODO add failsafe reports
+                launchable("record tests --session ${launchableSession} maven './**/target/surefire-reports' './**/target/failsafe-reports'")
```

Comment:
The Launchable CLI Maven test recorder implicitly adds `TEST-*.xml` as a suffix, so it is not needed here. It does not support comma-separated Ant-style globbing, but it does support multiple source roots as space-separated positional arguments.